pkg/testing/integration/program.go

```diff
@@ -2075,7 +2075,7 @@ func (pt *ProgramTester) prepareDotNetProject(projinfo *engine.Projinfo) error {
 		version := r.Replace(file)
 
 		err = pt.runCommand("dotnet-add-package",
-			[]string{dotNetBin, "add", "package", dep, "-v", version, "-s", localNuget}, cwd)
```

Comment:
I agree that this is probably the problem. It works locally on my Mac, so my initial suspicion is that `localNuget` points to the wrong place on the Mac test suite. But we use `localNuget` to get the version, so it must be correct. 

---

Either way, we can revert to unblock the release and then fix the test.

Reply:
I'm sorry about this - there's a drift between local dev env and CI env and my recent work makes it a bit worse even to hit some other objectives. Thank you for your patience.